code/datums/diseases/critical.dm

```diff
@@ -133,4 +138,65 @@
 					affected_mob.emote(pick("twitch", "gasp"))
 				if(prob(5) && ishuman(affected_mob))
 					var/mob/living/carbon/human/H = affected_mob
-					H.set_heartattack(TRUE)
\ No newline at end of file
+					H.set_heartattack(TRUE)
+
+/datum/disease/critical/hypoglycemia
+	name = "Hypoglycemia"
+	form = "Medical Emergency"
+	max_stages = 3
+	spread_flags = SPECIAL
+	spread_text = "The patient has low blood sugar."
+	cure_text = "Eating or glucose treatment"
```

Comment:
Glucose treatment will might make people think they need to inject saline glucose.
Having the cure text be as clear as can be is best.

Reply:
Saline glucose should cure it too otherwise it's a massive oversight that makes no sense